public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -99,12 +103,11 @@ export class LokiDatasource extends DataSourceApi<LokiQuery, LokiOptions> {
       }));
 
     for (const target of filteredTargets) {
-      // In explore we want to show result of metrics instant query in a table under the graph panel to mimic behaviour of prometheus.
-      // We don't want to do that in dashboards though as user would have to pick the correct data frame.
-      if (options.app === CoreApp.Explore && isMetricsQuery(target.expr)) {
+      if (target.instant) {
```

Comment:
I was thinking about adding an `else if` here to check ti we have url which doesn't have `instant` and `range` params and in that case  for `if (options.app === CoreApp.Explore && isMetricsQuery(target.expr))` run both queries, but I think this would be overkill would complicate a code. Basically now, if we have a link from older version of Grafana, we will run only a range query.

Reply:
Yeah I think that makes sense.